src/data/screenshots.json

```diff
@@ -9,8 +9,8 @@
         },
         "content": {
             "textblock": [
-                "Browse through the screenshots to get an overview of the features offered by the Corona-Warn-App with the release of Version 3.2 (available since 19. April, 2023). Please note that the screenshots may vary slightly between Android and iOS due to technical reasons.",
-                "With this update we are fixing bugs in the app. It does not contain any new features.",
+                "Browse through the screenshots to get an overview of the features offered by the Corona-Warn-App with the release of Version 3.2 (available since April, 19 2023). Please note that the screenshots may vary slightly between Android and iOS due to technical reasons.",
```

Comment:
"April, 19 2023" > "April 19, 2023"

the comma is in the wrong place

Reply:
Sorry, I fixed that. I shouldn't do edits on my phone :D

Reply:
@Ein-Tim 

> Sorry, I fixed that. I shouldn't do edits on my phone :D

It's usually better to have a bigger screen and this is not an urgent change, so wait until you have the right equipment available.